specs/core/1_shard-data-chains.md

```diff
@@ -148,6 +155,31 @@ class ExtendedShardBlockCore(Container):
     attester_bitfield: Bitvector[MAX_PERSISTENT_COMMITTEE_SIZE * 2]
 ```
 
+### `ShardState`
+
+```python
+class ShardState(Container):
+    history_acc: Vector[Hash, 64]
+    earlier_committee_rewards: List[uint64, MAX_PERSISTENT_COMMITTEE_SIZE]
```

Comment:
Is it in `Gwei`, and IIRC, we decided to keep all values unsigned?
```suggestion
    earlier_committee_rewards: List[Gwei, MAX_PERSISTENT_COMMITTEE_SIZE]
```

Reply:
The rewards are not denominated in gwei, they're unitless numerators of a fraction whose denominator is REWARD_COEFFICIENT_BASE.